source/robot_model/README.md

```diff
@@ -24,23 +24,23 @@ At creation, it uses the parser from `pinocchio` to read the `URDF` file and fil
 
 ## Robot kinematics
 
-Most common robotics functionalities are implemented such `forward_geometry`, `inverse_geometry` (in development),
-`forward_kinematic`, `inverse_kinematic`, ...
+Most common robotics functionalities are implemented such `forward_kinematics`, `inverse_kinematics`,
+`forward_velocity, `inverse_velocity`, ...
 
 ```cpp
-// forward geometry: pose of a robot frame from the robot joint positions
+// forward kinematics: pose of a robot frame from the robot joint positions
```

Comment:
I know you mention below that the end-effector is default, but I think it's not correct to write `pose of A robot frame` here because its the pose of the end-effector for all these examples